


  
<style> th { padding: 7px;} .markdown-body { max-width:1400px} </style>

Bellow the subsets and their corresponding features are presented.  
**Subset 1**  
  

|ID|Name|Last Update|Modality|Format|Size|Parent|Purpose|Link|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|1|Medication raw|2/26/2021|Files|CSV|279|0|Detailed information on all ordinations of ATC-codes: J01", "J02", "J04", "J05", "A07", "R02", "L03", "L04", "H02","L01"||
  
  
  
*Features of Subset 1*  

|Name|ID|Date of Intro|Values|Meaning of Nan|Meaning of Zero|Meaning of blankvoid|Parents|Unit|Definition|Purpose|Encoding|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|Timestamp|1|2/10/2021|date|date missing|Zero means no treatment with drug within period|Not possible in format.||YYYY-MM-DD|Start date and end date of drug ordination|Used to calculate length of treatment at calculate "time to fecal sample". (Obs.: Can be messy and double ordinations may occur. This is mostly solved by cleaning script).||
|Data on drug|2|2/10/2021|Many|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||Different|name, ATC-code, dosis, way of administration of specific drug|Used to label drugs||
|Ordination-ID|3|2/10/2021|0..inf; integer|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||none|Each ordination has a specific ID called Best./ord.-ID|Used to identify each ordination||
|Patient-ID|4|2/10/2021|0..inf; integer|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||none|Pseudonymized patient identifier|Used to identify each patient||
  
  
**Subset 2**  
  

|ID|Name|Last Update|Modality|Format|Size|Parent|Purpose|Link|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|2|Medication cleaned|2/26/2021|Files|CSV|279|1|Cleaned mediacl data where timeperiod and drugs of interest can be defined.||
  
  
  
*Features of Subset 2*  

|Name|ID|Date of Intro|Values|Meaning of Nan|Meaning of Zero|Meaning of blankvoid|Parents|Unit|Definition|Purpose|Encoding|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|Drug data|5|2/10/2021|many|No NA, NULL, or NONE|Zero means no treatment with drug within period|Not possible in format.||Different|Name of drug and days of treatment within desired interval prior to donation of fecal sample|Used to qualify and quantify treatment prior to fecal sample. ATC-codes avaliable: "J01", "J02", "J04", "J05", "A07", "R02", "L03", "L04", "H02","L01" (Antibiotics and immune modulators)||
|Sample_ID|6|2/10/2021|0..inf; integer|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||none|Each fecal sample has a specific ID|Used to map each feacl sample to patient ID and microbiome outcome||
  
  
**Subset 3**  
  

|ID|Name|Last Update|Modality|Format|Size|Parent|Purpose|Link|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|3|Microbiome_raw|1/10/2021|Files |Fastq|724|0|Raw data from illumina sequencing protocol||
  
  
  
*Features of Subset 3*  

|Name|ID|Date of Intro|Values|Meaning of Nan|Meaning of Zero|Meaning of blankvoid|Parents|Unit|Definition|Purpose|Encoding|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|Sample_ID|7|2/10/2021|0..inf; integer|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||none|Each fecal sample has a specific ID|Used to map each feacl sample to patient ID and microbiome outcome||
|Fastq files|8|2/10/2021|A T C or G|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||none|Output from illumina sequencing protocol|The basis of our microbiome research||
  
  
**Subset 4**  
  

|ID|Name|Last Update|Modality|Format|Size|Parent|Purpose|Link|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|4|PGP output|2/10/2021|Files|CSV|724|3|Files generated by running our PGP pipeline on fastq files||
  
  
  
*Features of Subset 4*  

|Name|ID|Date of Intro|Values|Meaning of Nan|Meaning of Zero|Meaning of blankvoid|Parents|Unit|Definition|Purpose|Encoding|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|Sample_ID|9|2/10/2021|0..inf; integer|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||none|Each fecal sample has a specific ID|Used to map each feacl sample to patient ID and microbiome outcome||
|Antibiotic resistance genes|4|2/10/2021|many|No NA, NULL, or NONE|Zero means gene not present|Not possible in format.||None|For each gene the number of mappings is normalized to the total number for 16s genes in the samples|Used to identify and quantify antibiotic resistance genes in each sample. Normalized to 16s genes to enable comparissons between samples.||
|ARG diversity and gene richness|5|2/10/2021|0..inf; integer|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||None|Diversity and gene richness calculated based on ARGs in each sample|Used to collapse all ARGs into overall measures comparable between samples.||
|Taxonomy|6|2/10/2021|many|No NA, NULL, or NONE|Zero means species not found|Not possible in format.||none|Mapping of reads to bacterial taxa. Normalized to genome length and total amount of non human reads in each sample|Used to identify and quantify bacterial composition of each sample.||
|Taxonomical diversity and richness|7|2/10/2021|0..inf; integer|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||none|Diversity and gene richness calculated based on bacterial species in each sample|Used to collapse all bacterial reads into overall measures comparable between samples.||
|Genetic functionality|8|2/10/2021|many|No NA, NULL, or NONE|Zero means functionality not found|Not possible in format.||none|Name of genetic functionality and number of mappings|Used to identify and quantify bacterial genetic functionalities||
  
  
**Subset 5**  
  

|ID|Name|Last Update|Modality|Format|Size|Parent|Purpose|Link|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|5|Clinical outcomes|2/10/2021|files|CSV|279|0|Detailed information on clinical outcomes from PERSIMUNE data lake||
  
  
  
*Features of Subset 5*  

|Name|ID|Date of Intro|Values|Meaning of Nan|Meaning of Zero|Meaning of blankvoid|Parents|Unit|Definition|Purpose|Encoding|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|Patient ID|9|2/10/2021|0..inf; integer|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||none|Pseudonymized patient identifier|Used to identify each patient||
|Sample_ID|10|2/10/2021|0..inf; integer|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||none|Each fecal sample has a specific ID|Used to map each feacl sample to patient ID and microbiome outcome||
|Conditioning|11|2/10/2021|MAC, MINE, UBC|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||none|Conditioning regiment|Conditioning regiment have impact on the microbiome and numerous other patient outcomes||
|GVHD|12|2/10/2021|Yes/no|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||none|Did the patient devellop graft vs. host disease|aGVHD is a servere complication to HSCT and propably has an extented interplay with the gutmircobiome trough components the immunesystem||
|Engraftment|13|2/10/2021|date|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||YYYY-MM-DD|The date of engraftment|The date of engraftment is when the immunesystem is reconstructed after HSCT. This is a curcial timepoint in understanding HSCT-patients.||
  
  
**Subset 6**  
  

|ID|Name|Last Update|Modality|Format|Size|Parent|Purpose|Link|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|6|Demographics|2/10/2021|files|CSV|279|0|data on social/societal, economic, sex, or historical medical events from LPR||
  
  
  
*Features of Subset 6*  

|Name|ID|Date of Intro|Values|Meaning of Nan|Meaning of Zero|Meaning of blankvoid|Parents|Unit|Definition|Purpose|Encoding|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|Age|15|2/10/2021|0..inf; integer|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||yrs|Age|Used to stratify for potential age specific traits ||
|Sex|16|2/10/2021|M/F|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||none|Gender|Used to stratify for potential gender specific traits ||
|Patient ID|17|2/10/2021|0..inf; integer|No NA, NULL, or NONE|Not possible in format.|Not possible in format.||none|Pseudonymized patient identifier|Used to identify each patient||
  
  
**Subset 7**  
  

|ID|Name|Last Update|Modality|Format|Size|Parent|Purpose|Link|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|7|Laboratory values|2/10/2021|files|CSV|279|0|Detailed information on laboratory values outcomes from PERSIMUNE data lake||
  
  
  
*Features of Subset 7*  

|Name|ID|Date of Intro|Values|Meaning of Nan|Meaning of Zero|Meaning of blankvoid|Parents|Unit|Definition|Purpose|Encoding|
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
|Laboratory values|14|2/10/2021|0..inf; integer|Laboratory value was not obtained|Value was zero|Not possible in format.||Different|Biochemical and microbiological outcomes|Many laboratory values will probably correlate to microbiomestatus. Some will be irrelevant.||
  
